browser/app/js/components/PolicyInput.js

```diff
@@ -25,10 +26,26 @@ class PolicyInput extends Component {
     dispatch(actions.setPolicies([]))
   }
 
+  canBeSubmitted() {
+    const policies =  this.props.policies
+    const prefix = this.props.currentBucket + "/" + this.prefix.value + '*'
+    const origPrefix = this.prefix.value === "*" ? "*" : this.prefix.value + "*"
+ 
+    return (this.prefix.value !== undefined) && !policies.some(function(o){return (o["prefix"] === prefix) || (o["prefix"] === origPrefix)})
+  }
+
   handlePolicySubmit(e) {
     e.preventDefault()
     const {web, dispatch} = this.props
 
+    if (!this.canBeSubmitted()) {
+      dispatch(actions.showAlert({
+        type: 'danger',
+        message: "This prefix already exists."
```

Comment:
"Policy already exists on this prefix, please choose a different prefix." - word it better.